src/_community/rules.md

```diff
@@ -46,14 +46,38 @@ Essentially, please take requests from community team members seriously and list
 something. If you have a problem with what a team member is asking you to do, we advise that you step away from the 
 situation and send a private message to another team member.
 
-#### 5) Keep all projects legal & appropriate
-
-If you need help with a project, you will not be provided with it if your project may break laws, breach a Terms of
-Service or EULA (belonging to any organisation, not just the platforms we're using), or be considered malicious or 
-inappropriate.
-
-Additionally, keep your code appropriate. Do not sexualize your code or use any slurs or other forms of derogatory
-language, either in the code itself or in the assets or filenames that make up your project.
+#### 5) Keep all projects legal, legitimate & appropriate
+
+While anyone may make use of the Quilt toolchain in accordance with the licenses applied to each project, all projects 
+developed or promoted in Quilt community spaces must meet the following criteria:
+
+* Projects must not be malicious - which includes (but is not limited to) malware, mods that include backdoors, or 
+  mods that harass specific people or groups of people (for example, by automatically kicking them from servers with 
+  the mod installed)
+* Projects must not be sexualised, hateful or otherwise inappropriate in nature, and this also applies to all code 
+  and project assets posted for any purpose
+* Projects must not break laws, including copyright laws - this includes piracy tools, as well as projects that 
+  ignore or otherwise break the licenses of other projects
+* Projects must not break the Terms of Service or EULA of any organisation, including (but not limited to) Mojang, 
+  Microsoft, Discord and any other community platforms we use
```

Comment:
Might be worth noting youtube-dl as an example just to improve clarity. Its not hypothetical either, there's a fabric record mod for YouTube links

Reply:
Do people generally assume ytdl isn't often used for piracy? It's kinda hard to look at the number of porn sites it supports and not think that, IMO.

I feel like it's implicitly covered because of that, but I can clarify it if you think it'd be worth it

Reply:
youtube-dl *is* well known for "piracy" (serving content, outside of a visible iframe, is against their TOS) yes, it has systems to get around additional protection structures licensed content on the platform has, and I think the mentioned mod advertises this

Reply:
Resolved by incoming copyright section change